xbmc/addons/kodi-addon-dev-kit/include/kodi/libXBMC_pvr.h

```diff
@@ -13,7 +13,7 @@
 #include <string.h>
 #include <stdlib.h>
 #include <stdio.h>
-#include "xbmc_pvr_types.h"
+#include "addon-instance/PVRClient.h"
```

Comment:
There is also a PVRClient.h in the pvr codebase. Will two files with the same name not be confusing? Or maybe that doesn't matter.

Reply:
How about just `PVR.h`? No hard feelings about that, though.

Reply:
I like that.

Reply:
Have changed it to `#include "addon-instance/PVR.h"`, seems better for it :smile:, thought on begin to hold them with name equal to Kodi, to see `xbmc/pvr/addons/PVRClient.h` is the counterpart in Kodi about and do his work.